File: Leetcode/557. Reverse Words in a String III/solution.js
const reverseWords = (s) => {
  const sAr = s.split(" ")
  const reverseWords = []

  sAr.forEach((word) => {
    reverseWords.push(word.split("").reverse().join(""))
  })


  return reverseWords.join(" ")
}

console.log(reverseWords("Let's take LeetCode contest"))

File: Leetcode/557. Reverse Words in a String III/solution.rb
def reverse_words(s)
  splittAr = s.split(" ")
  reverseWords = []

  splittAr.each do |word|
    reverseWords.push(word.reverse)
  end

  return reverseWords.join(" ")
end

print reverse_words("Let's take LeetCode contest")
